ddln_sitl.proto

```diff
@@ -15,4 +16,6 @@ message UpdatePoseResponse {
 service SitlService {
     rpc SetOrigin(LatLongAlt) returns (Empty);
     rpc UpdatePose(UpdatePoseRequest) returns (UpdatePoseResponse);
+    rpc GetWorldInfo(Empty) returns (WorldInfo);
+    rpc WorldIntersection(WorldIntersectionRequest) returns (WorldIntersectionResponse);
```

Comment:
Add comment regarding coordinate system. It seems the position is internally handled a bit differently, e.g. coordinates with respect to world origin frame O instead of world frame W.

Reply:
Added comments specifying SITL reference frame.